src/main/java/de/feu/massim22/group3/agents/Desires/Desire.java

```diff
@@ -0,0 +1,27 @@
+package de.feu.massim22.group3.agents.Desires;
+
+import java.util.LinkedList;
+
+import de.feu.massim22.group3.agents.Desires.SubDesires.SubDesire;
+
+public abstract class Desire {
```

Comment:
Wir brauchen noch ein Feld in dem die "Kosten" (Schritte) gespeichert werden die zum Erfüllen des Desires benötigt werden

Reply:
Wofür brauchen wir das? Um abzuschätzen ob ein Task noch innerhalb der Deadline fertiggestellt werden kann?

Reply:
Der Supervisor soll ja entscheiden/priorisieren wer welchen Task bearbeitet. Für diese Entscheidung braucht er die Info.